git_machete/cli.py

```diff
@@ -75,38 +78,172 @@ def usage(command: str = None) -> None:
         """[1:])))
 
 
-def short_usage() -> None:
-    print(
-        fmt("<b>Usage: git machete [--debug] [-h] [-v|--verbose] [--version] "
-            "<command> [command-specific options] [command-specific argument]</b>"))
+def short_usage() -> str:
+    return (fmt("<b>Usage: git machete [--debug] [-h] [-v|--verbose] [--version] "
+                "<command> [command-specific options] [command-specific argument]</b>"))
 
 
 def version() -> None:
     print(f"git-machete version {__version__}")
 
 
+def create_cli_parser() -> argparse.ArgumentParser:
```

Comment:
Out of curiosity, is generation of shell completion scripts also supported by argparse? (Not that we strictly need that now since we've got hand-written bash&zsh completion)

Reply:
It doesn't seems so. There are some random libraries offering it, but not argparse itself from a quick research.